orchagent/portsorch.cpp

```diff
@@ -193,16 +196,32 @@ void PortsOrch::removeDefaultBridgePorts()
         throw "PortsOrch initialization failure";
     }
 
-    /* Remove bridge ports in default 1Q bridge */
-    for (size_t i = 0; i < attr.value.objlist.count; i++)
+    auto bridge_port_count = attr.value.objlist.count;
+
+    /* Remove SAI_BRIDGE_PORT_TYPE_PORT bridge ports in default 1Q bridge */
+    for (size_t i = 0; i < bridge_port_count; i++)
```

Comment:
`bridge_port_count` is `unsigned int`.
in `auto i = 0`, `i` will become `int`.
the compiler will complain about `error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]`.